api/api.py

```diff
@@ -87,9 +87,59 @@ def k8s_yaml(yaml: Union[str, List[str], Blob]) -> None:
   """
   pass
 
-def k8s_resource(name: str, yaml: Union[str, Blob] = "", image: Union[str, FastBuild] = "",
+def k8s_resource(workload: str, new_name: str = "",
+                 port_forwards: Union[str, int, List[int]] = [],
+                 extra_pod_selectors: Union[Dict[str, str], List[Dict[str, str]]] = []) -> None:
+  """Configures a kubernetes resources
+
+  If :meth:`k8s_resource_assembly_version` has been called with `1`, see
+  :meth:`k8s_resource_v1_DEPRECATED` instead.
+
+  Args:
+    workload: which workload's resource to configure. This is a colon-separated
+      string consisting of one or more of (name, kind, namespace, group), e.g.,
+      "redis", "redis:deployment", or "redis:deployment:default".
+      `k8s_resource` searches all loaded k8s workload objects for an object matching
+      all given fields. If there's exactly one, `k8s_resource` configures options for
+      that workload. If there's not exactly one, `k8s_resource` raises an error.
+      (e.g., "redis" suffices if there's only one object named "redis", but if
+      there's both a deployment and a cronjob named "redis", you'd need to specify
+      "redis:deployment").
+    new_name: if non-empty, will be used as the new name for this resource
+    port_forwards: Local ports to connect to the pod. If no
+      target port is specified, will use the first container port.
+      Example values: 9000 (connect localhost:9000 to the default container port),
```

Comment:
put these as bullets for readability?

Reply:
I would prefer this, but couldn't figure it out in 10 minutes.
(FWIW this is copy/pasted from the existing k8s_resource docs)